src/CMakeLists.txt

```diff
@@ -19,7 +19,9 @@ endif()
 
 if(BUILD_DOCUMENTATION)
   find_package(PythonInterp REQUIRED)
-
+  if(NOT PYTHONINTERP_FOUND)
+    message(SEND_ERROR "Building the documentation requires Python")
```

Comment:
@dzenanz A send error will continue processing.  I think line 28 needs amending as well.

perhaps
line #23:  add "no python interpreter found" to message.
line 28:  if( NOT PYTHONINTERP_FOUND  OR ...)

Reply:
I didn't realize that `SEND_ERROR` will continue processing. I will amend the commit soon.